src/data/testimonials.json

```diff
@@ -8,25 +29,53 @@
       "blogLink": "https://aniq55.github.io/"
     },
     {
-      "name": " Dhruv Apte",
-      "role": "KWoC '17 Mentor",
-      "quote": "I had a really good time at KWoC '17. My projects prospered well because of inputs from participants. Regular chaperoning from KOSS maintained that the quality of the coding program never touched alow point and maintained the coding frequency consistent throughout the program.",
-      "imageLink": "https://avatars.githubusercontent.com/the-ethan-hunt",
-      "blogLink": "https://the-ethan-hunt.github.io/about/"
+      "name": "Jigyasa Gupta",
+      "role": "KWoC '21 Student",
+      "quote": "KWoC was a great kickstart into the world of Open Source, which also taught various aspects of GitHub, understanding the codebase and making meaningful contributions as a beginner. Another worthy facade was learning the mannerisms of communicating with the mentors and fellow contributors who gave valuable insight and feedback.",
+      "imageLink": "https://avatars.githubusercontent.com/Jigyasa-G",
+      "blogLink": "https://medium.com/@jigyasatata_23607/kwoc-project-report-66a01c897f47"
     },
     {
-      "name": "Gautham Santhosh",
-      "role": "KWoC '17 Mentor",
-      "quote": "My project was a normal one before KWOC, something I had worked on in a hackathon. But I loved the idea but found it hard to do alone. But due to KWoC I got very good quality amazing contributors who helped to changed that small project into a top 3 product in ProductHunt and featured in many tech publications.",
-      "imageLink": "https://avatars.githubusercontent.com/gauthamzz",
-      "blogLink": "https://gauthamzz.github.io/"
+      "name": "Jessica John Britto",
+      "role": "KWoC '21 Student",
+      "quote": "Participating in KWOC has given me exposure to the open source community as well as a number of other open source programmes. I’ve also picked up a few more GitHub features and Git commands. It’s also given me the confidence to learn new tech tools/frameworks in a limited period of time that are relevant to the projects I’m interested in working on.",
+      "imageLink": "https://avatars.githubusercontent.com/JessicaJohnBritto",
+      "blogLink": "https://medium.com/@jessjohnbritto/kwoc-kharagpur-winter-of-code-project-report-4b7d852ec1e4"
     },
     {
-      "name": "Athitya Kumar",
-      "role": "KWoC '16 and '17' Mentor",
-      "quote": "The experience has been great both the years, with a lot of new students learning open-source development from KWoC by contributing to the projects. Some of my mentees have even gone ahead to do a GSoC project as well in the following summers, and are definitely are still actively contributing to open-source.",
-      "imageLink": "https://avatars.githubusercontent.com/athityakumar",
-      "blogLink": "https://athityakumar.github.io/blog/posts/Kharagpur_Winter_of_Code_2016/"
```

Comment:
Why remove old testimonials when the links are working? I find the old ones valuable as well.